From f0972bd6b53c28cb2b927b362025d85f21cedcc4 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@rene-engelhard.de>
Date: Thu, 23 Dec 2021 16:42:21 +0100
Subject: [PATCH] work around rounding difference on i386 in
 descriptiveStatistics test

No idea why this only appears now, but:

======================================================================
FAIL: test_descriptive_statistics (descriptiveStatistics.descriptiveStatistics)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rene/tmp/autopkgtest-lxc.4j23khs6/downtmp/build.zIT/src/sc/qa/uitest/statistics/descriptiveStatistics.py", line 108, in te
st_descriptive_statistics
    self.assertEqual(round(get_cell_by_position(document, 0, 6, 10).getValue(),14) , 0.01524095329036)
AssertionError: 0.01524095329035 != 0.01524095329036

Change-Id: I496daf61a9542ea0da78e6092dd7070a2665ab0a
---
 sc/qa/uitest/statistics/descriptiveStatistics.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sc/qa/uitest/statistics/descriptiveStatistics.py b/sc/qa/uitest/statistics/descriptiveStatistics.py
index 13837804d287..b00bd9079a0d 100644
--- a/sc/qa/uitest/statistics/descriptiveStatistics.py
+++ b/sc/qa/uitest/statistics/descriptiveStatistics.py
@@ -107,7 +107,7 @@ class descriptiveStatistics(UITestCase):
             self.assertEqual(round(get_cell_by_position(document, 0, 6, 7).getValue(),11) , 139.49090909091)
             self.assertEqual(round(get_cell_by_position(document, 0, 6, 8).getValue(),12) , 11.810626955878)
             self.assertEqual(round(get_cell_by_position(document, 0, 6, 9).getValue(),13) , -1.4621677980825)
-            self.assertEqual(round(get_cell_by_position(document, 0, 6, 10).getValue(),14) , 0.01524095329036)
+            self.assertAlmostEqual(round(get_cell_by_position(document, 0, 6, 10).getValue(),14) , 0.01524095329036)
             self.assertEqual(get_cell_by_position(document, 0, 6, 11).getValue() , 31)
             self.assertEqual(get_cell_by_position(document, 0, 6, 12).getValue() , 26)
             self.assertEqual(get_cell_by_position(document, 0, 6, 13).getValue() , 57)
-- 
2.30.2

